### PR TITLE
Update toggle and click handlers

### PR DESCRIPTION
**Pull request by klemens-kunz**
This PR covers two additions to the angular-ivh-treeview directive:
- Add an option to disable expand/collapse toggling on label click
- Add the ability to disable the click handler when only toggling expand/collapse

### CODE REVIEW

File: dist/ivh-treeview.js [hunk 1]
```diff
@@ -108,6 +108,36 @@ angular.module('ivh.treeview').directive('ivhTreeviewChildren', function() {
 
 
 /**
+ * Click logic for treeview nodes
+ *
+ * Handles node click on element click
+ *
+ * @private
+ * @package ivh.treeview
+ * @copyright 2015 NorthPage, Inc.
```
Comment (cgatesman) [inline dist/ivh-treeview.js]: What are the implications of this if it is contributed to an open source project?  Is this appropriate?

Comment (cgatesman) [inline dist/ivh-treeview.js]: I don't think copyrights mean much in open source: http://www.ifosslr.org/ifosslr/article/view/30

File: dist/ivh-treeview.js [hunk 2]
```diff
@@ -181,86 +211,87 @@ angular.module('ivh.treeview').directive('ivhTreeviewToggle', [function() {
...
+        var $el = $compile(tpl)(scope)
+          , $container = $twistieContainers.eq(ix);
+
+        // Clean out global template and append the new one
+        $container.html('').append($el);
+      });
+
+    }
+  };
+}]);
```
Comment (cgatesman) [inline dist/ivh-treeview.js]: Might want to look closer why this is showing this as a change.  Probably wouldn't want this difference in our PR to the real ivh.treeview.

File: dist/ivh-treeview.js [hunk 3]
```diff
@@ -616,16 +661,16 @@ angular.module('ivh.treeview').directive('ivhTreeview', ['ivhTreeviewMgr', funct
...
+
+angular.module('ivh.treeview').filter('ivhTreeviewAsArray', function() {
+  'use strict';
+  return function(arr) {
+    if(!angular.isArray(arr) && angular.isObject(arr)) {
+      return [arr];
+    }
+    return arr;
+  };
+});
```
Comment (cgatesman) [inline dist/ivh-treeview.js]: Here is another place that appears as a bogus diff.

File: dist/ivh-treeview.js [hunk 4]
```diff
@@ -694,62 +739,62 @@ angular.module('ivh.treeview').factory('ivhTreeviewBfs', ['ivhTreeviewOptions',
...
+
+          // Call the post-linking function, if any
+          if(link && link.post) {
+            link.post.apply(null, arguments);
+          }
+        }
+      };
+    }
+  };
+}]);
```
Comment (cgatesman) [inline dist/ivh-treeview.js]: Another noisy bogus diff.

**Comment by cgatesman** [comment]
:+1:

File: dist/ivh-treeview.js (hunk 2 above)
Comment (klemens-kunz) [inline dist/ivh-treeview.js]: There seems to be an inconsistency in the line endings in the original code. I think most of the code is using Windows line endings, but this twistie directive for example uses Unix line endings. My generated `ivh-treeview.js` file is using Windows line endings, which is why this is showing up as a diff here.

File: dist/ivh-treeview.js (hunk 1 above)
Comment (jwinfield) [inline dist/ivh-treeview.js]: Please remove this copyright

**Comment by jwinfield** [comment]
:+1:

**Pull request merged**